### DISCUSSION TITLE
Video DownloadHelper BETA 9.0.2.3a1 (beta) not seeing video to download on youtube - for Private streamed videos.

### DISCUSSION THREAD

**Discussion by chris0144** [aclap-dev/video-downloadhelper#888] [Help and Q&A]
### Your question

Video DownloadHelper BETA 9.0.2.3a1 (beta) is now not seeing videos to download on youtube - for Private streamed videos.  Public all good,Example: https://www.youtube.com/live/PN0gTFMpD5E

### Addon & CoApp version and details

Video DownloadHelper BETA 9.0.2.3a1 (beta).
Found companion app: VdhCoApp 2.0.19

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I am finding some other YouTube content that is not being recognized in VDH:

https://www.youtube.com/live/JW-eWRVVFNw
https://www.youtube.com/live/i2ET8V0hXkU

Those were randomly chosen off your mentoring video page, which also is not being recognized by VDH.  It does not appear to be private.  I was able to play it in the web page.

But others seem to be no problem:

https://www.youtube.com/watch?v=9Y-0nWVdBH4
https://www.youtube.com/watch?v=rI8hS6IystA

Is YouTube rolling out another in their long line of frivolous changes that upset VDH?

**Reply by GrampaWildWilly** [reply]
I just read a Reuters article reporting some new YouTube feature about adding notes to videos.  This is supposed to start rolling out today.  Maybe this is causing a problem for VDH.  Only @paulrouget Paul can determine if that's true.

**Comment by diegodlh** [comment]
> Video DownloadHelper BETA 9.0.2.3a1 (beta) is now not seeing videos to download on youtube - for Private streamed videos.

Maybe related to #501?

**Reply by GrampaWildWilly** [reply]
9.0.2.3a1 is kind of old now.  The current beta is 9.1.0.41.  But it does not cure the problem of YouTube videos that require a login.  That appears to include private videos, age-restricted videos, & purchased/rented videos.  I suspect purchased/rented videos may be protected by DRM anyway, so they would be a lost cause.  The others might be possible to get.  We still need @paulrouget Paul to tell us what his investigations have found, assuming he's investigated this issue.

In a post upthread here from me a couple of months ago I listed 4 YouTube videos, 2 that VDH failed to recognize & 2 that were no problem for VDH.  This has not changed.
